README.md

```diff
@@ -14,18 +14,35 @@ files for assignments that require GatorGradle. ConfiGator uses
 
 ## Usage
 
-### Installing Python dependencies
+### Installing ConfiGator
 
-In order to install the dependencies of this project, run this command in your
-command line:
+Command to install ConfiGator
 
 ```bash
-poetry install
+pipx install configator
 ```
 
-After installing the dependencies, navigate to the repository directory, and run
-the following in your command line:
+### Command to Change Specific Configurations
+
+The command below will show a list of every available command to change
+specific configurations in `config/gatorgrader.yml`, or `build.gradle`
+within a file of your choice.
```

Comment:
```suggestion
specific configurations in `config/gatorgrader.yml` or `build.gradle`.
```

Reply:
`within a file of your choice` is redundant here; you already list the files.

Reply:
By the way, you can easily accept this suggestion with the **Commit suggestion** button above.